src/main/java/com/company/URLDecoder.java

```diff
@@ -29,14 +31,17 @@ public void readFromFile() {
 
 
     // Decodes a shortened URL to its original URL.
-    public void decode(String shortUrl) {
+    public String decode(String shortUrl) {
         for (int i = 0; i < urls.size(); i++) {
-            String str = urls.get(i).split(" = ", 2)[0];
-            if (shortUrl.equals(str)) {
-                System.out.println(urls.get(i).split(" = ", 2)[1]);
+            if(urls.contains(shortUrl)) {
```

Comment:
This is not what you need to do.
You either iterate and check or search for index and use it.